Driverless AI/natural-language-processing-tutorial-sentiment-analysis /natural-language-processing-tutorial-sentiment-analysis.md

```diff
@@ -424,17 +465,38 @@ A Bi-directional GRU model is like putting two independent RNN models in one. Ta
 
      In Driverless AI, we pass word embeddings as input to CNN models, get cross-validated predictions from it and use them as a new set of features.
 
-     -  **Character Embeddings**
+     - **Character embeddings**
 
-     Natural language processing is complex as the language is hard to understand given small data and different languages.Targeting languages like Japanese, Chinese where characters play a major role, we have character level embeddings in our recipe as well.
+     Natural language processing is complex as the language is hard to understand given small data and different languages. Targeting languages like Japanese, Chinese where characters play a major role, we have character level embeddings in our recipe as well.
 
      In character embeddings, each character gets represented in the form of vectors rather than words. Driverless AI uses character level embeddings as input to CNN models and later extract class probabilities to feed as features for downstream models. This gives the ability to work in languages other than English also. In case of languages like Japanese, Chinese etc where there is no concept of words, character embeddings will be useful.
 
+- **BERT/DistilBERT Models for Feature Engineering:**
+
+BERT and [DistilBERT](https://arxiv.org/abs/1910.01108) models can be used for generating embeddings for any text columns. These pre-trained models are used to get embeddings for the text followed by Linear/Logistic Regression to generate features that can then be used for any downstream models in Driverless AI.
+
+- **PyTorch Transformer Architecture Models (eg. BERT) as Modeling Algorithms:**
+
+Starting with Driverless AI 1.9 release, the Transformer-based architectures shown in the diagram below are supported as models in Driverless AI.
+
+![bert-models](assets/bert-models.jpg)
+
+The BERT model support multiple languages. [DistilBERT](https://arxiv.org/abs/1910.01108) is a distilled version of BERT that has fewer parameters compared to BERT (40% less) and it is faster (60% speedup) while retaining 95% of BERT level performance. The DistilBERT model can be useful when training time and model size is important
```

Comment:
@Castro38 should be: "The Best model supports multiple languages"

Reply:
@5675sp Done  - updated to "The BERT model supports..."